client/my-sites/marketing/index.js

```diff
@@ -20,7 +21,7 @@ export default function () {
 	} );
 
 	page( '/marketing/ultimate-traffic-guide*', function redirectToWPCoursesPage() {
-		window.location.replace( 'https://wordpress.com/learn/courses/intro-to-seo/' );
+		window.location.replace( localizeUrl( 'https://wordpress.com/support/courses/seo/' ) );
```

Comment:
Linter was complaining about wrapping this in localizeURL. Is this ok? @Automattic/i18n